documentatie/flowcharts/polling-station-state.md

```diff
@@ -0,0 +1,23 @@
+# Polling Station State
+
+This document describes the states a polling station can have.   
+The transition labels describe the endpoint that is used for performing the transition.
+The "save" endpoint, used to for [First/Second]EntryInProgress states is kept out, because Mermaid doesn't render transitions to itself too well.
+
+```mermaid
+stateDiagram
+NotStarted --> FirstEntryInProgress: delete
```

Comment:
```suggestion
NotStarted --> FirstEntryInProgress: claim
```

Reply:
Fixed in 99290e407f1b8717bce06e7ec53d7f0495a9f3da